vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -225,6 +225,9 @@ EXPORT(int, sceGxmColorSurfaceInit, SceGxmColorSurface *surface, SceGxmColorForm
     if (strideInPixels & 1)
         return RET_ERROR(SCE_GXM_ERROR_INVALID_ALIGNMENT);
 
+    if ((strideInPixels < width) || ((data.address() & 3) != 0))
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_VALUE);
```

Comment:
Shouldn't we return SCE_GXM_ERROR_INVALID_ALIGNMENT for data.address() & 3 case?

Reply:
Unless I misinterpreted something during RE, this is how it is supposed to be although I agree with you, SCE_GXM_ERROR_INVALID_ALIGNMENT  would be more appropriated for this case.